app/Models/ComponentGroup.php

```diff
@@ -23,13 +24,30 @@ class ComponentGroup extends Model implements HasPresenter
     use SearchableTrait, SortableTrait, ValidatingTrait;
 
     /**
+     * Viewable by public.
```

Comment:
Yes. That badly named annotation describes the type of constants and properties.